### DISCUSSION TITLE
Home assistant integration

### DISCUSSION THREAD

**Discussion by raidnet-ms** [AdguardTeam/AdGuardHome#3964] [Q&A]
Hello,
Where can I find the "home assistant" "adguard home" integration repository?

thank you

**Comment by ainar-g** [comment]
Hello.  Do you mean this one?  <https://github.com/hassio-addons/addon-adguard-home>  It's on our “Getting Started” page on the Wiki.

**Comment by raidnet-ms** [comment]
hi ainar,
this is the addon, somewhere should also be the integration, at least so I was been told when I posted the issue on the addon link that you reported.
Unfortunately with an answer with not many details...
